onnxruntime/core/platform/windows/env.cc

```diff
@@ -43,8 +40,6 @@ EXTERN_C IMAGE_DOS_HEADER __ImageBase;
 
 namespace onnxruntime {
 
-namespace {
```

Comment:
What is the reason this was removed?

Reply:
Because compiler complain this - cannot implement a class's functions in an anonymous namespace.